src/components/Tooltip/__stories__/tooltip.stories.mdx

```diff
@@ -125,57 +140,164 @@ Tooltip’s arrow can appear from top, bottom, left or right.
   </Story>
 </Canvas>
 
+### Themes
+
+Tooltip’s arrow can appear from top, bottom, left or right.
+
+<Canvas>
```

Comment:
CC @hadasfa WDYT about the following implementation?
```
<Canvas>
  <Story name="Themes">
    {Object.values(Tooltip.themes).map(theme => (
      <div className="monday-storybook-tooltip_top" key={theme}>
        <Tooltip
          modifiers={modifiers}
          content={capitalize(theme)}
          shouldShowOnMount
          position={Tooltip.positions.BOTTOM}
          animationType="expand"
          theme={theme}
        >
        <div/>
        </Tooltip>
      </div>
    ))}
  </Story>
</Canvas>
```

It's shorter , and will automatically support new themes. However, the code snipper doesn't look too good:
![image](https://user-images.githubusercontent.com/96776835/154032047-437c8e85-86cf-4168-ba1e-1bd9b97b6500.png)

I think it may be worth it, since this example will be used mainly to visually see all the themes, and not as a code reference. WDYT?

Reply:
Like it :)

Reply:
just aligned with the others..will change